src/constants.js

```diff
@@ -0,0 +1,6 @@
+export const map = {
+    SearchOptions: {
+        MAP: 'map',
+        LIST: 'list'
+    }
+};
```

Comment:
Added a new constants file that we can use for constants and enums that are for functionality but not text.